polaris-tokens/src/utilities.ts

```diff
@@ -118,7 +118,9 @@ export function getCustomPropertyNames(tokens: Tokens) {
     .flat();
 }
 
-export type BreakpointsTokenGroup = typeof breakpointsTokenGroup;
+export type BreakpointsTokenGroup = {
+  [key in keyof typeof breakpointsTokenGroup]: string | TokenGroup;
+};
```

Comment:
@aaronccasanova thoughts on opening this structure to either token structure (metaTokens or just tokens)?

Or do you think we should have two different types? Maybe something like the following:

```ts
type ExtractValues<T extends { value: string }> = {
  [key in keyof T]: T['value'];
type ExtractValues<T extends { value: string }> = {
  [key in keyof T]: T['value'];
}

export type BreakpointsMetaTokenGroup = typeof breakpointTokenGroup;
export type BreakpointsTokenGroup = ExtractValues<BreakpointMetaTokenGroup>;
```

Reply:
Alright, updated the types and moved removal of metadata to a utility function 👍 lmk wha you think @aaronccasanova